pulsar-client-admin-api/src/main/java/org/apache/pulsar/client/admin/Brokers.java

```diff
@@ -268,12 +268,12 @@
      *
      * @throws PulsarAdminException if the healthcheck fails.
      */
-    void healthcheck() throws PulsarAdminException;
+    void healthcheck(String topicVersion) throws PulsarAdminException;
```

Comment:
Maybe keep these old methods `healthcheck() and healthcheckAsync()` and add deprecated annotations? Then add the new ones.  Keeps the api compatibility.

Reply:
sure will keep them as deprecated

Reply:
done